### ISSUE TITLE
Cannot Open Any Podcasts in Repod

### ISSUE THREAD

**Issue by nate2014jatc** [thrillfall/nextcloud-gpodder#156]
Hello,

While using Repod on my Nextcloud 28 server, I am unable to open any podcast details. 
The error log seems to indicate that the issue is in the gpoddersync component, so I am filing an issue here.

Below is the error message that spawns when I try and load a podcast.

Issue also filed with Repod [HERE](https://git.crystalyx.net/Xefir/repod/issues/58)

```
[index] Error: Call to a member function get() on null in file '/var/www/nextcloud/apps/gpoddersync/lib/Core/PodcastData/PodcastDataReader.php' line 87
	GET /apps/repod/podcast?url=https%3A%2F%2Fwww.theguardian.com%2Faustralia-news%2Fseries%2Ffull-story%2Fpodcast.xml
	from 172.69.22.154 by Nate at Feb 3, 2024, 9:10:14 PM

```

PHP Info:
```
Version: 8.1.27
Memory limit: 1 GB
Max execution time: 3600
Upload max size: 128 MB
OPcache Revalidate Frequency: 2
Extensions: Core, date, libxml, openssl, pcre, zlib, filter, hash, json, Reflection, SPL, session, standard, sodium, apache2handler, apcu, mysqlnd, PDO, xml, bcmath, bz2, calendar, ctype, curl, dom, mbstring, FFI, fileinfo, ftp, gd, gettext, gmp, iconv, igbinary, imagick, intl, exif, mysqli, pdo_mysql, Phar, posix, readline, redis, shmop, SimpleXML, sockets, sysvmsg, sysvsem, sysvshm, tokenizer, xmlreader, xmlwriter, xsl, zip, Zend OPcache
```

DB Info:
```
Type: mysql (mariaDB)
Version: 10.5.21
Size: 276 MB
```

Host Info:
```
Debian 11 (bullseye) Container
```

**Comment by nate2014jatc** [comment]
[raw.log](https://github.com/thrillfall/nextcloud-gpodder/files/14155494/raw.log)

**Comment by nate2014jatc** [comment]
Issue resolved with the fix [HERE](https://git.crystalyx.net/Xefir/repod/issues/58#issuecomment-5302).

TLDR: Ensure that you have a cache configured, Repod is getting an update that should resolve the crash if there's not a cache configured.
